openc3/lib/openc3/packets/json_packet.rb

```diff
@@ -32,13 +32,21 @@ class JsonPacket
     attr_accessor :packet_time
     attr_accessor :stored
     attr_accessor :json_hash
+    attr_accessor :received_time
+    attr_accessor :extra
 
-    def initialize(cmd_or_tlm, target_name, packet_name, time_nsec_from_epoch, stored, json_data, key_map = nil)
+    def initialize(cmd_or_tlm, target_name, packet_name, time_nsec_from_epoch, stored, json_data, key_map = nil, received_time_nsec_from_epoch: nil, extra: nil)
       @cmd_or_tlm = cmd_or_tlm.intern
       @target_name = target_name
       @packet_name = packet_name
       @packet_time = ::Time.from_nsec_from_epoch(time_nsec_from_epoch)
       @stored = ConfigParser.handle_true_false(stored)
+      if received_time_nsec_from_epoch
+        @received_time = ::Time.from_nsec_from_epoch(received_time_nsec_from_epoch)
+      else
+        @received_time = @packet_time
```

Comment:
`@received_time` is new ... does anyone use this?

Reply:
It just makes JsonPacket more like Packet.